package.json

```diff
@@ -59,23 +71,41 @@
     "eslint-config-amex": "^6.0.0",
     "eslint-config-prettier": "^2.2.0",
     "eslint-plugin-prettier": "^2.1.2",
+    "husky": "^3.0.9",
     "import-glob-loader": "^1.1.0",
     "jest": "^21.2.1",
     "node-sass": "^4.5.0",
     "prettier": "^1.6.1",
+    "rc-progress": "^2.2.5",
     "react": "^16.0.0",
     "react-dom": "^16.0.0",
     "react-router-dom": "^4.0.0",
     "react-test-renderer": "^16.0.0",
     "react-transition-group": "^2.2.1",
-    "rc-progress": "^2.2.5",
     "rimraf": "^2.5.2",
     "sass-loader": "^6.0.2",
+    "semantic-release": "^15.13.31",
     "style-loader": "^0.13.2",
     "webpack": "^2.0.0",
     "webpack-dev-server": "^2.0.0"
   },
   "peerDependencies": {
     "react": "^15.0.0 || ^16.0.0"
+  },
+  "release": {
+    "plugins": [
+      "@semantic-release/commit-analyzer",
+      "@semantic-release/release-notes-generator",
+      "@semantic-release/changelog",
+      "@semantic-release/git",
+      "@semantic-release/npm"
```

Comment:
You need to flip the order of `/git` and `/npm` https://github.com/semantic-release/git#examples